packages/create-astro/src/actions/template.ts

```diff
@@ -93,7 +93,16 @@ export default async function copyTemplate(tmpl: string, ctx: Context) {
 				dir: '.',
 			});
 		} catch (err: any) {
-			fs.rmdirSync(ctx.cwd);
+			// Only remove the directory if it's most likely created by us.
+			if (ctx.cwd !== '.' && ctx.cwd !== './' && !ctx.cwd.startsWith('../')) {
+				try {
+					fs.rmdirSync(ctx.cwd);
+				} catch (_) {
```

Comment:
We could write the error in a `console.error`

Reply:
I'm on the fence about logging it. We'd only hit this codepath if something else goes wrong, so this particular error seems irrelevant (or at least, much less important than the main error).